auth/auth.go

```diff
@@ -291,31 +291,31 @@ func (auth *Authenticator) AuthenticateUser(username string, password string) Us
 
 // Authenticates a user based on a JWT token string and a set of providers.  Attempts to match the
 // issuer in the token with a provider.
-// Used to authenticate a JWT token coming from an insecure source (e.g. client request)
+// Used to authenticate a JWT token coming from an insecure source (e.g. § request)
```

Comment:
§?

Reply:
No idea how that made it in there! Fixed